Test/Unit/Helper/CartTest.php

```diff
@@ -2674,6 +2682,8 @@ public function getCartData_whenPaymentOnlyAndHasOrderPayload_returnsCartData()
         $quote->collectTotals()->save();
 
         TestUtils::setQuoteToSession($quote);
+
+        $sessionToMetadataSwitch = TestUtils::saveFeatureSwitch(Definitions::M2_ADD_SESSION_ID_TO_CART_METADATA);
```

Comment:
Should we run this (and other) tests with both options: with and without feature switch?

Reply:
We should. Will implement. Thanks.

Reply:
done